pypyr-ops/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# note ENTRYPOINT is still pypyr, meaning you can run this container as if it's
+# the pypyr binary.
+# be aware that the WORKDIR is NOT the GOPATH.
+# the WORKDIR is /src, which is the pypyr working dir. this will let you mount a
+# volume with a pipelines dir in it without hiding the go path.
+# build me like this from repo root:
+# docker build -t pypyr/pypyr-ops -t pypyr/pypyr-ops:latest -t pypyr/pypyr-ops:1.0.0 -f pypyr-ops/Dockerfile .
+
+FROM pypyr/pypyr-go:pypyr1.0.0-go1.11.1
+
+ARG terraformversion=0.11.8
+
+ARG awsversion=1.16.34
+
+RUN  apt-get update && apt-get install -y apt-transport-https; \
+        wget https://releases.hashicorp.com/terraform/0.11.8/terraform_${terraformversion}_linux_amd64.zip; \
```

Comment:
parameterize the terraform version `wget https://releases.hashicorp.com/terraform/${terraformversion}/terraform_${terraformversion}_linux_amd64.zip;`